packages/nocodb/src/helpers/catchError.ts

```
@@ -389,6 +389,18 @@ export function extractDBError(error): {
             table: extractTableNameMatch[1],
           };
         }
+
+        const extractColumnNameMatch = error.message.match(
+          / column "(\w+)" does not exist/i,
+        );
+
+        if (extractColumnNameMatch && extractColumnNameMatch[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)